### PR TITLE
Support Oculus Mobile SDK 1.5.0

### PR DESCRIPTION
**Pull request by liaxim**
GearVRf-DCO-1.0-Signed-off-by: Mihail Marinov <m.marinov@samsung.com>

### CODE REVIEW

**Comment by rahul27** [comment]
Looks good. Tested a few samples and it worked fine.

My only comment would be to rename the commit from 1.0.5 to 1.5.0 to avoid confusion about the version of the SDK.

**Comment by liaxim** [comment]
Fixed the incorrect version

**Comment by thomasflynn** [comment]
keep in mind we need to update any READMEs / wikis that reference the oculus version.

File: GVRf/Framework/backend_oculus/src/main/jni/ovr_activity.cpp [hunk 1]
```diff
@@ -196,14 +195,15 @@ void GVRActivity::onDrawFrame() {
 
     for ( int eye = 0; eye < VRAPI_FRAME_LAYER_EYE_MAX; eye++ )
     {
-        ovrFrameLayerTexture& eyeTexture = parms.Layers[VRAPI_FRAME_LAYER_TYPE_WORLD].Textures[eye];
+        ovrFrameLayerTexture& eyeTexture = parms.Layers[0].Textures[eye];
```
Comment (roshanch) [inline GVRf/Framework/backend_oculus/src/main/jni/ovr_activity.cpp]: we are always using 0th layer?

Comment (liaxim) [inline GVRf/Framework/backend_oculus/src/main/jni/ovr_activity.cpp]: We have one TimeWarp layer so 0 would be it.

**Comment by thomasflynn** [comment]
also, this sdk supposedly supports 64bit.  so, perhaps, we no longer need to strip out 64bit libs in the gradle files?

**Comment by liaxim** [comment]
Yes, I will test the 64bit support and update the gradle files.

**Comment by rahul27** [comment]
Can we add the 64 bit support in a separate pull request. I need this one for the controller support.

**Comment by liaxim** [comment]
Sure, I can have the 64bit related changes as a separate thing.

**Comment by liaxim** [comment]
Planning to merge Tuesday morning.

**Pull request merged**